### PR TITLE
[BENCH-1202]  - Support installing CLI version based on environment

### PR DESCRIPTION
**Pull request by rogerwangcs**
Read the `terra-server` metadata and install the CLI from the appropriate broad/verily distribution url. For verily, fetch the distribution path from afs, for broad, install from the open source repo.

### CODE REVIEW

File: service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh [hunk 1]
```diff
@@ -376,15 +376,43 @@ ${RUN_AS_LOGIN_USER} "\
...
-  curl -L https://github.com/DataBiosphere/terra-cli/releases/latest/download/download-install.sh | bash && \
-  cp terra '${TERRA_INSTALL_PATH}'"
+# Fetch the Terra CLI server environment from the metadata server to install appropriate CLI version
+readonly TERRA_SERVER="$(get_metadata_value "instance/attributes/terra-cli-server")"
+
+# If the server environment is a verily server, use the verily download script.
+# Otherwise, install the latest DataBiosphere CLI release.
+if [[ $TERRA_SERVER == *"verily"* ]]; then
+  # Map the CLI server to the appropriate AFS service environment
+  declare -A serverMap=(
```
Comment (ddietterich) [inline service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh:386]: You could use a case statement here. Then the default case can generate the error.

File: service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh [hunk 2]
```diff
@@ -376,15 +376,43 @@ ${RUN_AS_LOGIN_USER} "\
...
 
-${RUN_AS_LOGIN_USER} "\
-  curl -L https://github.com/DataBiosphere/terra-cli/releases/latest/download/download-install.sh | bash && \
-  cp terra '${TERRA_INSTALL_PATH}'"
+# Fetch the Terra CLI server environment from the metadata server to install appropriate CLI version
+readonly TERRA_SERVER="$(get_metadata_value "instance/attributes/terra-cli-server")"
+
+# If the server environment is a verily server, use the verily download script.
+# Otherwise, install the latest DataBiosphere CLI release.
+if [[ $TERRA_SERVER == *"verily"* ]]; then
```
Comment (ddietterich) [inline service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh:384]: Never seen that style of pattern match. There is another one, but if that works, that's cool

**Review by ddietterich — APPROVED** [review]
I'd prefer using a `case`, but this looks like it works

File: service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh (hunk 2 above)
Comment (ddietterich) [reply service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh:384]: There is a `=~` operator that matches a regex.

I didn't realize that the `==` operator would match a bash pattern. TIL

**Review by zloery — APPROVED** [review]

**Pull request merged**